elasticsearch5/Dockerfile

```diff
@@ -76,3 +77,5 @@ RUN  ln -s /etc/sv/elasticsearch /etc/service/
 RUN ln -sf /dev/stdout /var/log/elastic-scripts.log
```

Comment:
I don't have the context of where elastic-scripts was used. They were empty during my runs